test/SizedArray.jl

```diff
@@ -30,7 +31,7 @@
         @test @inferred(SizedArray{Tuple{2,2}}([1 2;3 4]))::SizedArray{Tuple{2,2},Int,2,2} == [1 2; 3 4]
         # From Array, reshaped
         @test @inferred(SizedArray{Tuple{2,2}}([1,2,3,4]))::SizedArray{Tuple{2,2},Int,2,1} == [1 3; 2 4]
-        @test_throws DimensionMismatch SizedArray{Tuple{4}}([1 2; 3 4])
+        @test_throws DimensionMismatch SizedArray{Tuple{1,4}}([1 2; 3 4])
```

Comment:
We can test the original `SizedArray{Tuple{4}}([1 2; 3 4])` now doesn't throw errors.
```suggestion
        @test SizedArray{Tuple{4}}([1 2; 3 4]) == vec([1 2; 3 4])
        @test_throws DimensionMismatch SizedArray{Tuple{1,4}}([1 2; 3 4])
```